docs/guides/features/icons.md

```diff
@@ -44,8 +61,18 @@ If you need to source your icons from some other location, you can edit this par
 
 :::info Note on filetypes
 
-- icon.icns = macOS
-- icon.ico = MS Windows
-- \*.png = Linux
+- `icon.icns` = macOS
+- `icon.ico` = Windows
+- `*.png` = Linux
+- `Square*Logo.png` & `StoreLogo.png` = Currently unused but intended for AppX/MS Store targets.
+
+Note that especially the png icons may be used on other platforms. Therefore we recommend including all icons even if you intend to only build for a subset of platforms.
```

Comment:
```suggestion
Note that the `png` icons may be used on other platforms. Therefore we recommend including all icons even if you intend to only build for a subset of platforms.
```

Reply:
i wanted to say that there is a possibility that the other types may be used outside their primary platform too, not just png

Reply:
How about that phrasing then (updated suggestion)?